_includes/snippets_library/ComputeCanada_Graham_slurm/13/print-sched-variables.snip

```diff
@@ -1,13 +1,29 @@
 > ## Job environment variables
 >
-> When SLURM runs a job, it sets a number of environment variables for the job. One of these will
-> let us check our work from the last problem. The `SLURM_CPUS_PER_TASK` variable is set to the
-> number of CPUs we requested with `-c`.
->
-> Using the `SLURM_CPUS_PER_TASK` variable, modify your job so that it prints how many CPUs have
-> been allocated.
+> When {{ site.sched.name }} runs a job, it sets a number of environment variables for the job.
+> One of these will let us check what directory our job script was submitted from.
+> The `SLURM_SUBMIT_DIR` variable is set to the directory from which our job was 
```

Comment:
The scheduler has been abstracted out to a liquid variable, but the actual environment variables in the text are still literals, and are SLURM-specific.

Reply:
Since this is a snippet in the ComputeCanada SLURM cluster library, I'm comfortable leaving these hard-coded. I'm skeptical that the variable for `SLURM_SUBMIT_DIR` would be used anywhere outside of this challenge problem, and don't want to litter the config files.